src/app/helpers/environmentVariablesHelper.js

```diff
@@ -7,7 +7,8 @@ let envVariables = {
   LEARNER_URL: env.sunbird_learner_player_url || 'https://staging.open-sunbird.org/api/',
   CONTENT_URL: env.sunbird_content_player_url || 'https://staging.open-sunbird.org/api/',
   CONFIG_URL: env.sunbird_config_service_url || 'https://staging.open-sunbird.org/api/config/',
-  CONFIG_CACHE_TTL: env.config_cache_ttl || 600,
+  CONFIG_REFRESH_INTERVAL: env.config_refresh_interval || 10,
+  CONFIG_SERVICE_ENABLED: env.config_service_enabled || true,
```

Comment:
by Default the value should be False not true

Reply:
As per Bajaj advice i set it to true,can we change it to false ?